core/file_walker.go

```diff
@@ -60,9 +60,12 @@ func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[s
 			files = append(files, file)
 		}
 	}
-	ret.Name = name
 	if parent != nil {
+		ret.Name = name
 		ret.Parent = parent
+	} else {
+		// Root dir
+		ret.Name = dirName + name
```

Comment:
Ehh, what?

Instead of `ret.Name = dirName + name` we could create "fake files" for `home` and `/` (as our "root" dir is `/home/leon`):
```go
// For sure we would create this automatically, just to make my point clear
ret.Parent = &File{Name: "home", Parent: &File{Name: "/"}}
```

`ret.Path()` would then still return `/home/leon/some_file`